database-plugins/src/test/java/co/cask/hydrator/plugin/teradata/test/TeradataPluginTestRun.java

```diff
@@ -358,56 +358,22 @@ public void testNonExistentDBTable() throws Exception {
 
     ETLBatchConfig etlConfig = new ETLBatchConfig("* * * * *", sourceBadName, table, transforms);
     Id.Application appId = Id.Application.from(Id.Namespace.DEFAULT, "dbSourceTest");
```

Comment:
will try and make these tests more foolproof. Really need a better way of cleaning up when using TestBase - https://issues.cask.co/browse/CDAP-4925